app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,25 @@
+class LikesController < ApplicationController
+  before_action :current_user, only: [:create]
+
+  def new
+    @like = Like.new
+  end
+
+  def create
+    @like = Like.new(likes_params)
+    @like.author_id = current_user.id
+    @like.post_id = params[:post_id]
+
+    if @like.save
+      redirect_to user_post_path(current_user.id, Post.find(params[:post_id]))
+    else
+      render new
+    end
+  end

```

Comment:
- [ ] I'm unable to add a like to a post. Please also make sure that a user should be able to add a like to a given post.

Reply:
I have added a link above demonstrating this feature